functions/_fzf_configure_bindings_help.fish

```diff
@@ -21,7 +21,7 @@ DESCRIPTION
     sequences. Rather, consider using fish_key_reader to manually validate them.
 
     In terms of experimentation, fzf_configure_bindings erases any bindings it previously installed
-    before installing new ones so it can be repeatedly invoked in the same fish session without
+    before installing new ones so it can be repeatedly executed in the same fish session without
```

Comment:
out of scope change: simply using consistent terminology of execute a function instead of invoke a function